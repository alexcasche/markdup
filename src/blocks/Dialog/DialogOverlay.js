import styled from 'styled-components'

const DialogOverlay = styled.div`
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  background-color: ${props => props.theme.colors.black};
  opacity: 0.5;
  z-index: -1;
`

export default DialogOverlay
