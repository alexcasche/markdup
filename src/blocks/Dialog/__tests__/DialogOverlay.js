import DialogOverlay from '../DialogOverlay'

describe('DialogOverlay Element', () => {
  it('renders correctly', () => {
    const tree = render(<DialogOverlay />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
