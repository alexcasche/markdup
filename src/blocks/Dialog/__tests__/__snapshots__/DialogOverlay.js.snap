// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogOverlay Element renders correctly 1`] = `
.c0 {
  position: fixed;
  top: 0;
  right: 0;
  bottom: 0;
  left: 0;
  background-color: #1D1F21;
  opacity: 0.5;
  z-index: -1;
}

<div
  className="c0"
/>
`;
